### PR TITLE
Reduce warnings in SDK build (for post-unified-docs SDK)

### PR DESCRIPTION
**Pull request by JoelPasvolsky**
Not to be deployed yet. Should fix build warnings on duplicate labels and intersphinx links.

### CODE REVIEW

**Comment by JoelPasvolsky** [comment]
osx-3.7 failure due to "Homebrew must be run under Ruby 2.6! You're running 2.3.3"

**Comment by m3ller** [comment]
@JoelPasvolsky, I am working to fix this issue that you are observing. After the `osx3.7 failure` issue is fixed, can this code be merged?

**Comment by JoelPasvolsky** [comment]
Thanks @m3ller, this should be merged just before we put out an updated dwave-ocean-sdk.

File: docs/license.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
 License
```
Comment (m3ller) [inline docs/license.rst]: Not sure about my knowledge on this, but do RST titles need to be framed with equal signs at the top AND at the bottom?
```
=======
License
=======
```

Also, I realize that my suggestion is NOT part of your PR, so feel free to ignore it.

Comment (JoelPasvolsky) [reply docs/license.rst]: Functionally no, but would be nice to be more consistent, and we have this mixed throughout. RST is quite flexible with marking headers.

Comment (m3ller) [reply docs/license.rst]: Cool! Thanks for letting me know. :)

**Pull request merged**